shoop/core/taxing/_tax_summary.py

```diff
@@ -16,7 +16,14 @@
 
 class TaxSummary(list):
     @classmethod
-    def from_line_taxes(cls, line_taxes, untaxed=TaxlessPrice(0)):
+    def from_line_taxes(cls, line_taxes, untaxed=None):
+        """
+        :type line_taxes: Iterable[LineTax]
+        :type untaxed: TaxlessPrice
+        :rtype: TaxSummary
+        """
+        if untaxed is None:
+            untaxed = TaxlessPrice(0)
```

Comment:
I created PR #10 for that, but it does not have your commit aa52149ee3397c1ef33692834a77970121c3c097. Maybe just drop the other commit from this PR?